test/utils.js

```diff
@@ -0,0 +1,48 @@
+var assert = require('chai').assert;
+var sass = require('node-sass');
+
+var render = function(options) {
+  options.outputStyle = 'compressed';
```

Comment:
Agreed.
On 17 May 2016 1:44 AM, "Kaelig" <notifications@github.com> wrote:

> In test/utils.js
> <https://github.com/kaelig/sass-project-boilerplate/pull/1#discussion_r63374835>
> :
>
> > @@ -0,0 +1,48 @@
> > +var assert = require('chai').assert;
> > +var sass = require('node-sass');
> > +
> > +var render = function(options) {
> > +  options.outputStyle = 'compressed';
>
> I'd vote for compressed, non-overridable, with a comment explaining why
> it's that way.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaelig/sass-project-boilerplate/pull/1/files/4c6935f6f309b5617eadc1d7c7f49d23f64a9ede#r63374835>
>